2 - ListsLogicAndIterations/Code/Dictionarys.cs

```diff
@@ -5,19 +5,40 @@ namespace ListsLogicAndIterations.Code
 {
 	public static class Dictionarys
 	{
-		public static Dictionary<Guid, decimal> Build(Guid key, decimal value)
-		{
-			throw new NotImplementedException();
+        public static Dictionary<Guid, decimal> Build(Guid key, decimal value)
+        {
+            Dictionary<Guid, decimal> dict = new Dictionary<Guid, decimal>
+            {
+                { key, value }
+            };
```

Comment:
This is a place where I would use `var` instead of it being totally optional. A lot of code can be cut out making it easier to "get to the point" of the assignment.

Other than that, its perfecto!